docker-for-mac/release-notes.md

```diff
@@ -13,6 +13,39 @@ This page contains information about the new features, improvements, known issue
 
 {% include eula.md %}
 
+## Docker Desktop 3.6.0
+2021-08-10
+
+[Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn}
+[Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn}
+
+### New
+
+- You can now create a Dev Environment from your local git repository
+- Add sorting to volume list
+- Add name search to volumes list
+
+### Upgrades
+
+- [Compose V2 RC1](https://github.com/docker/compose-cli/releases/tag/v2.0.0-rc.1)
+- Go runtime to 1.16.7
+- Docker Engine to v20.10.8
+- containerd to v1.4.9
+- runc v1.0.1
+- Kubernetes v1.21.3 
+- Golang 1.16.6
+
+### Bug fixes and minor changes
+
+- Fixed bug where the DNS server would fail after receiving an unexpectedly large datagram.
+- Fixed spurious traces on iptables updates
+- Fixed slowness when adding multiple ports forwarding option
+- Fixed an issue related to hardware acceleration [docker/for-mac#5121](https://github.com/docker/for-mac/issues/5121)
+- Fixed an issue related to Skip This Update for mac (https://github.com/docker/for-mac/issues/5842)
+- Do not automatically switch CLI contexts on application start or when switching between Windows and Linux containers. Fixes [docker/for-mac#5787](https://github.com/docker/for-mac/issues/5787).
+- Fixed connection leak when using `virtualization.framework` and not using `vpnkit`.
+
+
 ## Docker Desktop 3.5.2
 2021-07-08
 
```

Comment:
update the link to the correct build here: https://github.com/docker/docker.github.io/pull/13321/files#diff-c8a46636ff5669abbabd1223c466929322c319dca9a6c5794114f35dae973187L19-L20

```
[Download for Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/66501/Docker.dmg)
[Download for Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/66501/Docker.dmg)
```